contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,317 @@
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Flutter:** Product analytics and feature flags only
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does PostHog cost?
+
+PostHog has [transparent pricing]((/pricing)) based on the usage. It’s free to get started and completely free for the first 1 million A/B testing requests. After this free monthly allowance, you'll pay $0.0001/request, and PostHog charges progressively less the more you use. You can also set billing limits to ensure you don't get surprise bills. 
+
+While PostHog is open source, its A/B testing features aren't included in the open source release.
+
+> **Further reading:** New to A/B testing? Read [a software engineer's guide to A/B testing](/blog/ab-testing-guide-for-engineers) and our guide to [common A/B testing mistakes](/blog/ab-testing-mistakes).
+
+## 2. VWO
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-green text-lg">✔</span>
+- **Get started without talking to sales:** <span className="text-red text-lg">✖</span>
+- **Best for:** Non-technical teams
+
+![VWO mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/vwo-testing.png)
+
+### What is VWO?
+
+[VWO](https://vwo.com/) is a testing platform that primarily targets large enterprises. Its experimentation platform includes support for A/B and multivariate tests and a visual editor. It's useful for non-technical users who need an easy-to-use interface to test in-app messaging and UI copy.
+
+They also offer product analytics, so you don't need a separate platform or integration to link your analytics data to your A/B test results.
+
+### Which platforms does VWO support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **Flutter:** <span className="text-green text-lg">✔</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does VWO cost?
+
+VWO mobile app A/B tests starts at $1,526 per month for up to 10k users. This is billed annually, so it means an upfront cost of $18,312. Also note that this price is only for mobile A/B tests. If you want to include analytics in your package, prices start at an additional $15,936 per year.
+
+Lastly, if you also want to run server-side tests, you also need to purchases a separate package. Prices for this start at $28,164 per year.
+
+## 3. Optimizely
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
+- **Open source:** <span className="text-red text-lg">✖</span>
```

Comment:
Their SDKs are open source, and some templates: https://github.com/optimizely (same amount of open source as VWO)